components/centraldashboard/frontend/main-page.html

```diff
@@ -116,7 +126,7 @@
                 <aside class="flex"></aside>
                 <footer class="footer">
                     <section class='privacy'>Privacy</section>
-                    <section class='build'>build version v1.0</section>
+                    <section class='build'>build version v[[buildVersion]]</section>
                 </footer>
             </app-drawer>
             <app-header-layout fullbleed>
```

Comment:
GitHub won't let me comment on the actual code a few lines below, but is anything using the `NamespaceSelector` at this time? Similar to the `Activity` tab, I don't think we should show it if it doesn't currently do anything. Fine to leave the code, but maybe hide the element?

Reply:
Good idea, I will hide it behind an app-scoped variable.

... And done!

![image](https://user-images.githubusercontent.com/5303018/52690867-e850bc00-2f13-11e9-86bf-417d3856050d.png)